templates/zerver/emails/missed_message.html

```diff
@@ -12,7 +12,7 @@
             {% endfor %}
         {% endfor %}
     {% else %}
-    <div class="missed_message">
+    <div class="missed_message no_content">
```

Comment:
I added this class because we want to apply CSS to the `<a>` tags inside this `<div>`. However, we do not want to apply CSS to the `div.missed_message` above, as it contains dynamic content.

Reply:
Makes sense; renamed this to `no-content-explanation`.